impl/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -19,3 +19,6 @@ kafka-streams.internal.leave.group.on.close=true
 # Deactivate exposure of metrics through JMX beans
 # It is still adding a mxBean in AppInfoParser though
 kafka-streams.auto.include.jmx.reporter=false
+# For compatibility with generic decorators and Quarkus versions prior to 3.11.0
+# TODO: remove in main branch as the problem is not reproducible with Quarkus 3.11.0 and later versions
+quarkus.test.flat-class-path=true
```

Comment:
do you need this property since you have switched to bean injection?

Reply:
removed